.github/workflows/build-m1-binaries.yml

```diff
@@ -15,88 +15,6 @@ on:
 env:
   CHANNEL: "nightly"
 jobs:
-  build_wheels:
```

Comment:
why is build_wheels job removed from GHA?

Reply:
It is old one. In this PR, we switch to the latest workflow for the M1 Wheel. https://github.com/pytorch/audio/pull/2907/files#diff-3dab8d7b6418e0047d1bdd73ae68c53d57e16fa2972a16060b86636140c3fb23L41-L44